### ISSUE TITLE
Emulator auto-upgrade issue - no such file or directory

### ISSUE THREAD

**Issue by vishwacsena** [Microsoft/BotFramework-Emulator#1343]
Hit this issue when I clicked on check for updates in Emulator. 

Emulator version prior to upgrade: 4.1.45083. 

Note: After this error, Emulator seems to have successfully downloaded the upgrade package and was able to install it via 'reboot to update' option. 

![image](https://user-images.githubusercontent.com/20918136/53828307-21d48000-3f32-11e9-9b91-8efe1f645215.png)

**Comment by tonyanziano** [comment]
So is this the correct sequence of events?:

1. Checked for update
2. Displayed error
3. Went back to Help menu and the option changed to "Restart to update" which you then clicked and was successful?

**Comment by vishwacsena** [comment]
@tonyanziano - that's correct. Also saw this on my Mac OS.

**Comment by tonyanziano** [comment]
Ok thanks, I'll check it out.

**Comment by corinagum** [comment]
FYI I'm reproing 

![image](https://user-images.githubusercontent.com/14900841/53912529-fa52e580-400d-11e9-93cd-aa3ce36b219d.png)
My version was a nightly: 4.2.x
![image](https://user-images.githubusercontent.com/14900841/53912575-15255a00-400e-11e9-8633-223b485632aa.png)

**Comment by tonyanziano** [comment]
I have been trying to repro it and was only able to successfully repro once after deleting the `__update__` folder within the app data folder.

I looked at the stack trace and it just seems to be an issue inside of the electron updater, but it doesn't end up stopping the update process, and I was able to successfully update after dismissing the error.

Since it is caused by a dependency and doesn't lead to any issues, I'm thinking we should just swallow the error if we detect it.

Trying to get a more consistent repo though so I can test the behavior of a potential fix.
